_posts/en/2019-10-15-case-continuous-improvement.md

```diff
@@ -44,7 +44,7 @@ If that sounded a little math-y or jargon-y, we'll walk through it below.
 
 If we graph f(x) = x we will get a line as follows
 
-![Graph of f(x) = x]({{site.baseurl}}/assets/images/graphx.PNG)
+![Graph of f(x) = x](/ITStrategy/assets/images/graphx.PNG)
```

Comment:
@gabrielcossette Pourquoi avoir changé le chemin des images?

Reply:
Je me demande la même chose

Reply:
This commit: https://github.com/sara-sabr/ITStrategy/commit/5f666ca574e2ed85e9516854d7a77377b353e78f

Reply:
@gcharest @smellems Si je me rappelle bien, j'avais une erreur dans les test de URLs... Ce nouveau chemin fonctionne aussi bien dans la page et est plus facile à retenir ;)

Reply:
Le build passe sur master, donc ça ne devrait pas être un problème.

La variable site.baseurl équivalent à `/ITStrategy`, donc la même chose.

Dans le build il y a des avertissements pour des liens qui ne peuvent pas être testé, mais le build passe quand même

![image](https://user-images.githubusercontent.com/2353299/71842744-67049980-3090-11ea-977b-bdde9a5447c8.png)

Je crois que ça vas être pareil si on enlève la variable.  J'aimerais mieux conserver la variable, comme ça si on bouge notre site ça devrait continuer de fonctionner.

Reply:
Je suis d'accord, c'est préférable de garder la variable, surtout dans l'éventualité où on finirait par avoir notre propre domaine.

Reply:
D'après moi, juste mettre "/ITStrategy" va fonctionner même avec un nouveau domaine et nous n'aurions plus de "Warning". Je vais faire d'autres tests pour valider je proposerai les changements dans un autre PR.